lib/Models/Catalog/CatalogItems/I3SCatalogItem.ts

```diff
@@ -0,0 +1,98 @@
+import i18next from "i18next";
+import {
+  computed,
+  makeObservable,
+  observable,
+  override,
+  runInAction,
+  toJS
+} from "mobx";
+import BoundingSphere from "terriajs-cesium/Source/Core/BoundingSphere";
+import Cartesian2 from "terriajs-cesium/Source/Core/Cartesian2";
+import isDefined from "../../../Core/isDefined";
+import I3SCatalogItemTraits from "../../../Traits/TraitsClasses/I3SCatalogItemTraits";
+import CreateModel from "../../Definition/CreateModel";
+import { ModelConstructorParameters } from "../../Definition/Model";
+import { CatalogMemberMixin, MappableMixin } from "terriajs-plugin-api";
+import UrlMixin from "../../../ModelMixins/UrlMixin";
+import I3SDataProvider from "terriajs-cesium/Source/Scene/I3SDataProvider";
+import { getName } from "../../../ModelMixins/CatalogMemberMixin";
+import ArcGISTiledElevationTerrainProvider from "terriajs-cesium/Source/Core/ArcGISTiledElevationTerrainProvider";
+import { createCesium3DTileStyle } from "../../../ModelMixins/Cesium3dTilesMixin";
+
+export default class I3SCatalogItem extends MappableMixin(
+  UrlMixin(CatalogMemberMixin(CreateModel(I3SCatalogItemTraits)))
+) {
+  static readonly type = "I3S";
+  readonly type = I3SCatalogItem.type;
+
+  @observable
+  private dataProvider?: I3SDataProvider;
+  public boundingSphere: BoundingSphere | undefined;
+
+  constructor(...args: ModelConstructorParameters) {
+    super(...args);
+    makeObservable(this);
+  }
+
+  async forceLoadMapItems() {
+    if (!isDefined(this.url)) {
+      throw `\`url\` is not defined for ${getName(this)}`;
+    }
+    this.dataProvider = await I3SDataProvider.fromUrl(this.url, {
+      geoidTiledTerrainProvider: this.terrainURL
+        ? await ArcGISTiledElevationTerrainProvider.fromUrl(this.terrainURL)
+        : undefined
+    });
+    this.boundingSphere = BoundingSphere.fromBoundingSpheres(
+      this.dataProvider.layers
+        .map((layer) => layer.tileset?.boundingSphere)
+        .filter(isDefined)
+    );
```

Comment:
Nitpick: This could be split off as a `@computed get boundingSphere()` accessor. Not much difference either way, but one pro is that it separates out the logic for computing the bounding sphere.